### PR TITLE
Add the option to add default tags.

### PR DESCRIPTION
**Pull request by bviolier**
Adds the ability to set default tags when creating the exporter.

### CODE REVIEW

File: lightstepoc/options.go [hunk 1]
```diff
@@ -60,6 +60,15 @@ func WithComponentName(componentName string) Option {
 	}
 }
 
+// WithDefaultTags sets tags that will be appended to every span that is exported to the trace.
```
Comment (iredelmeier) [inline lightstepoc/options.go]: I think we should rename this to `WithDefaultAttributes` to be consistent with OpenCensus's naming. ("Tag" means something else to them; "attribute" is more akin to OpenTracing's tags.)

File: lightstepoc/options.go [hunk 2]
```diff
@@ -60,6 +60,15 @@ func WithComponentName(componentName string) Option {
...
+func WithDefaultTags(tags opentracing.Tags) Option {
+	return func(c *config) {
+		for key, value := range tags {
```
Comment (iredelmeier) [inline lightstepoc/options.go]: I think we should rename this to WithDefaultAttributes to be consistent with OpenCensus's naming. ("Tag" means something else to them; "attribute" is more akin to OpenTracing's tags.)

**Comment by iredelmeier** [comment]
Thanks @bviolier! This looks great to me aside from the suggested naming change.

"Attributes" aren't really clarified in the OC spec, so here's the `SpanData` type: https://github.com/census-instrumentation/opencensus-go/blob/aad2c527c5defcf89b5afab7f37274304195a6b2/trace/export.go#L73-L97

**Comment by bviolier** [comment]
Thanks, that makes sense! I tried to actually use trace.Attribute now, but the values are private, so I can't read them back to put them in Tags, unfortunately. See updated PR, let me know if this is ok!

**Pull request merged**